README.md

```diff
@@ -45,7 +49,7 @@ To use this sample form:
 1. Import the linked CRF into Rave for a subject of your choosing.
 2. Log in with the sample app using the credentials of the subject you chose.
 
-> You should see two forms, Form 1 and Form 2. Form 1 opens as one page. Form 2 opens as multiple pages.
+> As general user sees an Image Capture form that opens a form for user to take a picture or load an image from library. As Rave registered user, addition to ImageCapture form you should see two more forms, Form 1 and Form 2. Form 1 opens as one page. Form 2 opens as multiple pages.
```

Comment:
I think this probably belongs in a separate section, so we'd have:

- Using the Case Report Form (CRF)

and then

- Using the self-registration and data capture functionality

and the latter would have steps to register a user and upload data.